### PR TITLE
GDB-8419: Snowflake Account identifier is not properly mapped in the URL

### PR DESCRIPTION
**Pull request by boyan-tonchev**
## What
The url of Snowflake driver is not populate properly.

## Why
When jdbc driver is Snowflake then url has different url template than other drivers and the generic calculation of url is not applicable to it.

## How
Implemented different calculation function specific for Snowflake.

### CODE REVIEW

File: src/js/angular/repositories/ontop-repo.directive.js [hunk 1]
```diff
@@ -419,7 +445,7 @@ function ontopRepoDirective($uibModal, RepositoriesRestService, toastr, Upload,
         };
 
         const validateHostName = () => {
-            if ($scope.selectedDriver.driverType !== $scope.genericDriverType) {
+            if ($scope.selectedDriver.driverType !== JdbcDriverType.GENERIC) {
```
Comment (svilenvelikov) [inline src/js/angular/repositories/ontop-repo.directive.js:448]: Can you create a method like `isGenericDriver(type)` and reuse it as this check is performed too often here.
The method can be in the OntopDriverData class.

Comment (svilenvelikov) [reply src/js/angular/repositories/ontop-repo.directive.js:448]: Then you can also expose a boolean property isGenericDriver which to be used in the template where this check is also performed many times.

Comment (boyan-tonchev) [reply src/js/angular/repositories/ontop-repo.directive.js]: Done

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Review by svilenvelikov — DISMISSED** [review]

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Comment by boyan-tonchev** [comment]
Jenkins retry build

**Pull request merged**